proto/tendermint/statesync/types.proto

```diff
@@ -9,6 +11,8 @@ message Message {
     SnapshotsResponse snapshots_response = 2;
     ChunkRequest      chunk_request      = 3;
     ChunkResponse     chunk_response     = 4;
+    LightBlockRequest   light_block_request = 5;
```

Comment:
needs formatting

Reply:
Is there some proto lint command I can run to do this for me?

Reply:
make proto-format should do the job.